[2019-05-25 00:00:00.386] [INFO] console - 现在时间： 2019-05-24T16:00:00.386Z
[2019-05-25 08:34:40.993] [ERROR] console - Api-URL:http://172.31.15.184:8082//lottery/getawarddata--->{"lotterycode":"gxkl10"}---------------error
[2019-05-25 08:34:40.994] [ERROR] console - Error: timeout of 10000ms exceeded
    at createError (E:\dev-001\xq_eycp\Eycp.SitePC\node_modules\axios\lib\core\createError.js:16:15)
    at Timeout.handleRequestTimeout [as _onTimeout] (E:\dev-001\xq_eycp\Eycp.SitePC\node_modules\axios\lib\adapters\http.js:216:16)
    at ontimeout (timers.js:475:11)
    at tryOnTimeout (timers.js:310:5)
    at Timer.listOnTimeout (timers.js:270:5)
[2019-05-25 09:33:57.196] [ERROR] console - Api-URL:http://localhost:35129/home/GetAdData--->{"codes":"eycp_home_banner"}---------------error
[2019-05-25 09:33:57.198] [ERROR] console - Error: connect ECONNREFUSED 127.0.0.1:35129
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
[2019-05-25 10:36:35.220] [ERROR] console - Api-URL:http://localhost:35129/home/GetAdData--->{"codes":"eycp_home_banner"}---------------error
[2019-05-25 10:36:35.221] [ERROR] console - Error: connect ECONNREFUSED 127.0.0.1:35129
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
[2019-05-25 10:44:50.991] [ERROR] console - Api-URL:http://localhost:35129/home/GetAdData--->{"codes":"eycp_home_banner"}---------------error
[2019-05-25 10:44:50.992] [ERROR] console - Error: connect ECONNREFUSED 127.0.0.1:35129
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
[2019-05-25 10:44:55.895] [ERROR] console - Api-URL:http://localhost:35129/home/GetAdData--->{"codes":"eycp_home_banner"}---------------error
[2019-05-25 10:44:55.896] [ERROR] console - Error: connect ECONNREFUSED 127.0.0.1:35129
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
[2019-05-25 11:27:43.563] [INFO] console - Server running at:http://iZj6c74lqepn0tZ:8038
